compiler/src/iree/compiler/Codegen/Common/GPU/GPUReuseSharedMemoryAllocs.cpp

```diff
@@ -92,6 +92,7 @@ static LogicalResult populateLivenessRanges(
     }
   }
   if (sharedMemAllocs.size() < 2) {
+    LLVM_DEBUG(llvm::dbgs() << "shared mem allocs are less than 2.\n");
```

Comment:
I don't understand this comment. Does it mean that there are fewer than 2 shared memory allocations?

Reply:
yes .. reworded.